src/app/views/query-runner/request/Request.tsx

```diff
@@ -134,6 +134,18 @@ const Request = (props: any) => {
     dispatch(setDimensions(dimen));
   };
 
+  // Resizable element does not update it's size when the browser window is resized.
+  // This is a workaround to reset the height
+  window.addEventListener('resize', () => {
```

Comment:
We need to remove the event listener on unload

Reply:
Done. I am adding and disposing of it when the size of the screen changes between mobile and full screen. This takes care of the case where a user changes the browser size multiple times without reloading the page